package.json

```diff
@@ -24,7 +24,7 @@
     "tap": "^1.3.2"
   },
   "dependencies": {
-    "nan": "^2.0.5"
+    "nan": "^2.3.5"
```

Comment:
This has no purpose, we don't depend on any feature added in 2.3, and npm will always install the latest matching, and  `^2.0.5` matches `2.3.5`, so it will get installed.

Why did you think this necessary?